lib/action_cable/channel/streams.rb

```diff
@@ -12,7 +12,7 @@ module Channel
     #     def follow(data)
     #       stream_from "comments_for_#{data['recording_id']}"
```

Comment:
I have updated the second example in the stream docs to use `stream_for`. Is that what you were referring to, or do you think there needs to be an example dedicated to `stream_for`?